piqture/data_loader/mnist_data_loader.py

```diff
@@ -13,123 +13,130 @@
 from __future__ import annotations
 
 from functools import partial
-from typing import Union
+from typing import List, Optional, Tuple, Union
 
 import torch.utils.data
 import torchvision
 from torchvision import datasets
 
-from piqture.transforms import MinMaxNormalization
+from piqture.transforms.transforms import MinMaxNormalization
 
 
-def load_mnist_dataset(
-    img_size: Union[int, tuple[int, int]] = 28,
-    batch_size: int = None,
-    labels: list = None,
-    normalize_min: float = None,
-    normalize_max: float = None,
+def load_mnist_dataset(  # pylint: disable=R0913, R0914, W0012
+    img_size: Union[int, Tuple[int, int]] = 28,
+    batch_size_train: int = 64,
+    batch_size_test: int = 1000,
```

Comment:
Having separate `batch_size` arguments is good!

Reply:
However, any particular reason behind these arguments having default values of 64 and 1000?